src/data/links.json

```diff
@@ -60,6 +60,10 @@
           "name": "Speakers' Dinner",
           "path": "/speakers-dinner"
         },
+        {
+          "name": "PyGameZero Workshop (Wednesday)",
+          "path": "/pygamezero"
```

Comment:
Again: Please move this up. This is not a social event.  I asked for this to be moved accordingly before.

```
- Conference events by day, i.e. events that happen once, during the
conference, such as workshop, summit, etc.
- Social events by day, i.e. events that happen once outside of the main
conference.
- Multiple events / events that span several days, such as the PyLdadies
programme, hackathon, etc.
```